storage/client/queue/src/samples/java/com/azure/storage/queue/javadoc/QueueJavaDocCodeSamples.java

```diff
@@ -86,7 +86,8 @@ public QueueClient createClientWithCredential() {
         QueueClient queueClient = new QueueClientBuilder()
             .endpoint("https://${accountName}.queue.core.windows.net")
             .queueName("myqueue")
-            .credential(SASTokenCredential.fromQuery("{SASTokenQueryParams}"))
+            .credential(SASTokenCredential.fromQueryParameters(
```

Comment:
Let's simplify these examples and assume that `SASTokenCredential.fromSASTokenString` is used. There will be more changes to SASTokenCredential creation once Files and Queues have the ability to generate their own SAS tokens.